Server/routing/index.js

```diff
@@ -27,39 +27,45 @@ router.get('/', async (ctx) => {
 
 router.get('/t/:miniURL', async (ctx) => {
     // find url in DB or error- does not exist
+    let id = parseInt(ctx.params.miniURL, 36);
+    console.log(id);
+    try {
+        var query = await database.query(
```

Comment:
Use `const` instead of `var` or, if you do actually reassign that variable later, use `let`. The old `var` keyword makes really weird decisions with regard to [variable scoping](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/let), and its behavior is almost never what we actually want.